sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -549,3 +552,33 @@ def service_properties_deserialize(generated: Any) -> Dict[str, Any]:
         'minute_metrics': Metrics._from_generated(generated.minute_metrics),  # pylint: disable=protected-access
         'cors': [CorsRule._from_generated(cors) for cors in generated.cors],  # pylint: disable=protected-access
     }
+
+class StorageConfiguration(Configuration):
+    """The home for all configurable policies in the pipeline,
+    including Storage specific policies.
+
+    :param polling_interval: Polling interval while doing LRO operations, if Retry-After is not set.
+    """
+
+    headers_policy: str
```

Comment:
Should only need to document that extra stuff we have that is not in `Configuration` (not 100% on whether you did that or not).

Also looks like there are some missing things in here? We should document everything we set here at least:
https://github.com/Azure/azure-sdk-for-python/blob/fa77a41bf294c9d965ce25f31b46edfdd9398355/sdk/storage/azure-storage-queue/azure/storage/queue/_shared/base_client.py#L419-L438